File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A2/026/026.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    string names[n];
    int weights[n];

    for (int i = 0; i < n; i++) {
        cin >> names[i] >> weights[i];
    }

    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - i - 1; j++) {
            if (weights[j] < weights[j + 1]) {
                swap(weights[j], weights[j + 1]);
                swap(names[j], names[j + 1]);
            }
        }
    }

    int more_than_15 = 0;
    for (int i = 0; i < n; i++) {
        if (weights[i] > 15) {
            more_than_15++;
        }
    }

    cout << more_than_15 << endl;
    cout << names[0] << " " << weights[0] << endl;

    return 0;
}

File: TOI-Zero_68/A2/026/026.py
n = int(input())

names = []
weights = []

for _ in range(n):
    name, weight = input().split()
    names.append(name)
    weights.append(int(weight))

for i in range(n - 1):
    for j in range(n - i - 1):
        if weights[j] < weights[j + 1]:
            weights[j], weights[j + 1] = weights[j + 1], weights[j]
            names[j], names[j + 1] = names[j + 1], names[j]

more_than_15 = sum(1 for weight in weights if weight > 15)

print(more_than_15)
print(names[0], weights[0])